backend/candidate_comparison_tool/service.py

```
@@ -0,0 +1,59 @@
+    """Send resume and JD to Gemini and return structured score breakdown as JSON.
+
+    Raise GeminiError on failure.
+    """
+    try:
+        model = genai.GenerativeModel(model_name="models/gemini-1.5-flash")
+        prompt = build_prompt(resume_text, job_description)
+        response = model.generate_content(prompt)
+        logger.debug("Raw Gemini response: %s", response.text)
+        return extract_json(response.text)
+    except Exception as e:
+        raise BaseGeminiError(original_exc=e) from e
```

Pylint (3.3.7)

[error] No value for argument 'message' in constructor call

(E1120)